data/attributes.yml

```diff
@@ -30209,6 +30209,36 @@ base_attributes:
   values:
   - vehicle_fluid_type__concentrate
   - vehicle_fluid_type__prediluted
+- id: 3415
+  name: Caffeine content
+  description: Indicates the amount of caffeine present in the coffee product, aiding consumers in choosing based on their desired caffeine intake level
+  friendly_id: caffeine_content
+  handle: caffeine-content
+  values:
+  - vcaffeine_content__regular
```

Comment:
This should be `caffeine_content__regular`.

Reply:
fixed